internal/integration/marshaler/marshaler.go

```diff
@@ -287,9 +287,24 @@ func jsonv3MarshalTxAckEvent(msg *integration.TxAckEvent) ([]byte, error) {
 		DeviceName:      msg.DeviceName,
 		FCnt:            msg.FCnt,
 		Tags:            msg.Tags,
+		TXInfo: models.TXAckTxInfo{
```

Comment:
To be honest, I would not mind if we only implement this in the Protobuf messages. The `json_v3` exist for backwards compatibility. I would like to encourage people to migrate to either the `protobuf` or `json` marshaler as `json_v3` will be removed at the next major release.

Reply:
Yes, you are right
i have added because there is already `jsonv3MarshalTxAckEvent` event for `json_v3`
You can remove

Thanks for review